client/test-data/corsica.locoflex.rococo.json

```diff
@@ -0,0 +1,120 @@
+{
+  "type": "FeatureCollection",
+  "community": {
+    "meta": {
```

Comment:
not setting a nominal income will set it to zero and testing will be restricted to accounts with native balance

Reply:
demurrage seems to have a reasonable default value, but setting it explicitly may still be helpful as means of documentation/example

Reply:
To be honest, I did not think to much about it, I simply took the `test-locations-mediterranean.json`, and adapted the existing values, but I agree that these missing values are not ideal.

Reply:
Ahh, I did not know that the ceremony rewards are 0 because the default in the chain specs is 1.